content/pcengines/buy/seabios/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: "Dasharo (coreboot+SeaBIOS) Entry Subscription for PC Engines Redirect"
```

Comment:
@Stanislaw-bnk leaving ` Entry Subscription` is intended?

Reply:
Fixed: d0ed34fa0ca94bb990f35b39c162238eae1ca698

Reply:
@artur-rs Any success with the `ERROR deprecated` SEO CI Fail?